client/state/sites/plans/selectors/is-current-plan-expired.ts

```diff
@@ -0,0 +1,15 @@
+import moment from 'moment';
+import { getCurrentPlan } from 'calypso/state/sites/plans/selectors/get-current-plan';
+import type { IAppState } from 'calypso/state/types';
+import type { SiteId } from 'calypso/types';
+
+export function isCurrentPlanExpired( state: IAppState, siteId: SiteId ): boolean | null {
+	const currentPlan = getCurrentPlan( state, siteId );
+
+	if ( ! currentPlan || ! currentPlan.expiryDate ) {
+		return null;
+	}
+
+	const expiration = moment( currentPlan.expiryDate ).startOf( 'day' );
+	return expiration < moment();
+}
```

Comment:
Similarly. This could fit under `/packages/data-stores/src/plans/hooks`
although not sure about importing `moment`, if not already imported. it's possible to do the logic with `Date` construct.

Reply:
Sounds good! I had assumed `moment` was used to parse a non-standard date format, but it turns out that the expiry date is returned in ISO 8601 format, which can be parsed by `Date()` in all browsers. (fbhepr%2Skers%2Sjcpbz%2Sjc%2Qpbagrag%2Snqzva%2Qcyhtvaf%2Sjcpbz%2Qovyyvat%2Sfgber%2Qcebqhpg%2Qyvfg.cuc%3Se%3Q6283np92%232591-og)